### DISCUSSION TITLE
Stackblitz

### DISCUSSION THREAD

**Discussion by brokuka** [unjs/community#39] [Q&A] [answered]
Giget fetch not working in stackblitz, any idea how to workaround ?

**Comment by pi0** [comment]
Can you please make a reproduction?

Unless there is a CORS issue with remote source it should work

**Reply by brokuka** [reply]
For reproduction my case, you need do some actions.

1) Create blank node js template
2) Delete all files
3) In terminal download template from github (`npx @brokuka/nuxt-starter`)
4) Go to the end promt (when template starting download)

**Reply by pi0** [reply]
Ah yes that's a CORS issue. You can reproduce it by literally running `fetch('https://api.github.com/repos/brokuka/nuxt-templates/tarball/v3-unocss')` in devtools.



<img width="1512" alt="image" src="https://github.com/user-attachments/assets/8bc7642d-3130-40b3-a36f-90ff8ed02760">

**Reply by pi0** [reply, marked as answer]
We might actually fix it!

**Reply by brokuka** [reply]
> We might actually fix it!

Appreciate that!
